doc/ch-run.rst

```diff
@@ -130,7 +135,8 @@ mounting SquashFS images with FUSE.
     Unset environment variables whose names match :code:`GLOB`.
 
   :code:`-v`, :code:`--verbose`
-    Be more verbose (can be repeated).
+    Print extra chatter; can be repeated. See the :ref:`FAQ entry on verbosity
+    <faq_verbosity>` for details.
```

Comment:
I discovered recently that you don't have to repeat the target title; you can just say:

```
:ref:`faq_verbosity`
```

and it will use the title of the target for the link title.